cropper/app/controllers/Application.scala

```diff
@@ -44,52 +50,30 @@ object Application extends Controller with ArgoHelpers {
 
   def index = Authenticated { indexResponse }
 
+  def export = Authenticated.async(parse.json) { httpRequest =>
+    httpRequest.body.validate[ExportRequest] map { exportRequest =>
```

Comment:
I believe `validate` is like `asOpt` but returning a more detailed error if failed.